docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -510,92 +492,96 @@ Ensure the `bytes32` value set under the permissions are correct according to th
 
-- **key:** `0x4b80742de2bfc6dd6b3c0000<address>`
-- **possible values:**
-  - `[ <target-contract-address>, 0x.... ]`: an **ABI-encoded** array of `address[]` defining the allowed addresses.
-  - `0x` (empty): if the value is an **empty byte** (= `0x`), the caller `<address>` is allowed to interact with any address (**= all addresses are whitelisted**).
+These contracts MUST implement the [ERC165](https://eips.ethereum.org/EIPS/eip-165) standard to be able to detect their interfaces.
 
-```json
-{
-  "name": "AddressPermissions:AllowedAddresses:<address>",
-  "key": "0x4b80742de2bfc6dd6b3c0000<address>",
-  "keyType": "MappingWithGrouping",
-  "valueType": "address[]",
-  "valueContent": "Address"
-}
-```
+|   Interface ID   |                     Meaning                      |
+| :--------------: | :----------------------------------------------: |
+|   `0xffffffff`   |    Interaction with any standard is allowed.     |
+| Other interfaces | Interaction with a specific standard is allowed. |
 
-![LSP6 Allowed Addresses explained](/img/standards/lsp6/lsp6-allowed-addresses.jpeg)
+</details>
 
----
+<details>
+    <summary>Addresses</summary>
 
-### Allowed functions
+|                   Address                    |                     Meaning                     |
+| :------------------------------------------: | :---------------------------------------------: |
+| `0xffffffffffffffffffffffffffffffffffffffff` |    Interaction with any address is allowed.     |
+|               Other addresses                | Interaction with a specific address is allowed. |
 
-You can also restrict which functions a specific address can run by providing a list of `bytes4` function selectors.
+</details>
 
-To restrict an `<address>` to only execute the function `transfer(address,uint256)` (selector: `a9059cbb`), the following key-value pair can be set in the ERC725Y contract storage.
+<details>
+    <summary>Functions</summary>
 
-- **key:** `0x4b80742de2bf8efea1e80000<address>`
-- **possible values:**
-  - `[ 0xa9059cbb, 0x... ]`: an **ABI-encoded** array of `bytes4[]` values, definiting the function selectors allowed.
-  - `0x` (empty): if the value is an **empty byte** (= `0x`), the caller `<address>` is allowed to execute any function (**= all `bytes4` function selectors are whitelisted**).
+|    Function     |                     Meaning                      |
+| :-------------: | :----------------------------------------------: |
+|  `0xffffffff`   |    Interaction with any function is allowed.     |
+| Other functions | Interaction with a specific function is allowed. |
 
-```json
-{
-  "name": "AddressPermissions:AllowedFunctions:<address>",
-  "key": "0x4b80742de2bf8efea1e80000<address>",
-  "keyType": "MappingWithGrouping",
-  "valueType": "bytes4[]",
-  "valueContent": "Bytes4"
-}
-```
+</details>
 
-:::info
+To restrict an `<address>` to be allowed to execute any function at this address `0xCA41e4ea94c8fA99889c8EA2c8948768cBaf4bc0` which should
```

Comment:
Restrict + allow in the same sentence looks a bit strange imo. Maybe we can break it down in bullet points? Not sure

```suggestion
To restrict an `<address>` to:
- interact only with the LSP0ERC725Account (ERC165 interface ID = ``0x66767497`) deployed at address `0xCA41e4ea94c8fA99889c8EA2c8948768cBaf4bc0`, 
- but allow to call any functions on this contract,

the data key - value pair below can be set in the ERC725Y contract storage.
```

Reply:
"To allow an `<address>` to execute any function at this address `0xCA41e4ea94c8fA99889c8EA2c8948768cBaf4bc0` which should
be an LSP0 standard `0x66767497`, the data key - value pair below can be set in the ERC725Y contract storage."

Wdyt?

Reply:
@b00ste 

To allow an `<address>` to execute any function on a LSP0ERC725Account (interface ID `0x66767497`) deployed at address `0xCA41e4ea94c8fA99889c8EA2c8948768cBaf4bc0`, the data key - value pair below can be set in the ERC725Y contract storage.